end-to-end-test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+./node_exporter-0.17.0.linux-amd64/node_exporter &
```

Comment:
You should really clean up these processes

Reply:
It's only really meant for use in CI right now. I wonder if there's an easy way to spawn all this.

Reply:
Ok, added some cleanup to the script. I'm thinking about moving the tool downloads into the script as well. But, incremental improvements.